CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ If you are running with a high `-ruler.num-workers` and if you're not able to ex
 Further, if you're using the configs service, we've upgraded the migration library and this requires some manual intervention. See full
 instructions below to upgrade your Postgres.
 
+* [CHANGE] Remove unnecessary `frontend.cache-split-interval` in favor of `querier.split-queries-by-interval` both to reduce configuration complexity and guarantee alignment of these two configs. #2040
```

Comment:
`frontend.cache-split-interval` default was `24h`, while `querier.split-queries-by-interval` default is `0` (which means disabled).

Reply:
That logic has actually been moved [here](https://github.com/cortexproject/cortex/pull/2040/files#diff-be5bce86bb962a94af90c800ee891d85R97-R109). This PR takes a similar approach to how the deprecation of `split-by-day` was implemented (in favor of `split-by-interval`) [here](https://github.com/cortexproject/cortex/blob/master/pkg/querier/queryrange/roundtrip.go#L99-L105).